packages/ipfs-cpinner-provider/src/IpfsPinnerProvider.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Content, CID, DID, IpfsPinner, Key, MetadataManager, IpfsClient
+} from './types'
+
+export default class IpfsPinnerProvider {
+  constructor (
+    private ipfsClient: IpfsClient,
+    private metadataManager: MetadataManager,
+    private ipfsPinner: IpfsPinner
+  ) {}
+
+  async put (did: DID, key: Key, content: Content): Promise<CID> {
```

Comment:
`put` or `post` or `add` or `create`?

- `post` comes from the HTTPS method that will use it
- `put` comes from IPFS
- `add` comes from who knows
- `craete` from CRUD

Reply:
You're right, at this level we should use CRUD acronym, this is the interface that the future API will be expecting. We should not use IPFS nor HTTP related statements. I'm changing it to CRUD